### PR TITLE
[ui][google-calendar] Use aliased calendar name 1.6.x

### PR DESCRIPTION
**Pull request by asmigala**
Google Calendar tests now add a random suffix to the calendar name to
prevent collisions between concurrent tests

//test: `@google-calendar`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by asmigala** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

File: ui-tests/src/test/java/io/syndesis/qe/steps/other/GoogleCalendarSteps.java [hunk 1]
```diff
@@ -38,11 +52,31 @@ public void fillInUpdateEventFormUsingCalendarSummaryAndDescription(String calen
...
+        for (List<String> row : data.cells()) {
+            List<String> newRow = new ArrayList<>(row);
+            newCells.add(newRow);
+            if("Calendar name".equals(newRow.get(0))) {
+                newRow.set(1, gcu.getAliasedCalendarName(newRow.get(1)));
+            }
+        }
+        // oh well, things are never as nice as you want them to be
+        DataTable newData = DataTable.create(newCells,
+                new DataTableTypeRegistryTableConverter(new DataTableTypeRegistry(Locale.getDefault())));
```
Comment (avano) [inline ui-tests/src/test/java/io/syndesis/qe/steps/other/GoogleCalendarSteps.java]: https://github.com/syndesisio/syndesis-qe/blob/ea5532ed683523ef6000c9d1f603cbba00eb4674/utilities/src/main/java/io/syndesis/qe/utils/DatatableUtils.java#L20-L27

:smile:

Comment (asmigala) [reply ui-tests/src/test/java/io/syndesis/qe/steps/other/GoogleCalendarSteps.java]: *facepalm*

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by asmigala** [comment]
```com.google.api.client.googleapis.json.GoogleJsonResponseException: 403 Forbidden
{
  "code" : 403,
  "errors" : [ {
    "domain" : "usageLimits",
    "message" : "Calendar usage limits exceeded.",
    "reason" : "quotaExceeded"
  } ],
  "message" : "Calendar usage limits exceeded."
}``` 

Well, fuck

**Comment by asmigala** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by asmigala** [comment]
The failures are due to the open issue syndesisio/syndesis#4021 , ready to merge

**Pull request merged**